src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const updatedArray = sourceString
+    .split(';')
+    .map((item) => {
+      return item.split(':');
+    })
+    .filter((item) => item.length > 1)
+    .map((item) => {
+      return [item[0].trim(), item[1].trim()];
+    });
```

Comment:
[CODE KNOWLEDGE] - Don't need to make a copy before each operation. Methods like `filter` or `map` don't mutate the original array, so you can chain them directly without assigning to a new variable at each step.